components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.ts

```diff
@@ -305,4 +327,79 @@ export class PolicyAddMembersComponent implements OnInit, OnDestroy {
     this.addOrRemoveQueuedMember(true, member);
     this.closeModal();
   }
+
+  private showInputs(inputName: string): void {
```

Comment:
Consider changing `inputName` to `fieldName`; I think that makes the intent more clear.